src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -160,15 +168,16 @@ private void handleExit() {
         primaryStage.hide();
     }
 
-    public PersonListPanel getPersonListPanel() {
-        return personListPanel;
+    public EntityListPanel getListPanel() {
+        return listPanel;
     }
 
     /**
      * Executes the command and returns the result.
      *
      * @see seedu.address.logic.Logic#execute(String)
      */
+    @SuppressWarnings("checkstyle:Regexp")
```

Comment:
Hey. Can I just check why this annotation is necessary?

Reply:
I think I might have accidentally suppressed checkstyle for that to make my build pass.